news/1668.bugfix

```diff
@@ -0,0 +1 @@
+updates the Makefile to ensure consistent installations of black and click. @Akshat2Jain
```

Comment:
Use the issue number in the file name. Use the internal extension.

Reply:
Sorry but I'm not familiar with that term . what exactly you meant by 'internal extension.'

Reply:
In the same way that we have the `.bugfix`, we also have the `.internal`. The options are here:

https://github.com/plone/plone.restapi/blob/master/pyproject.toml

We use `.internal`, when the change is interest only to the package developers.

Reply:
@Akshat2Jain see also https://6.docs.plone.org/contributing/index.html#change-log-entry

Reply:
there is no issue , so should I name the file as `1668.internal`